Python/Product/PythonTools/PythonTools/Repl/DebugReplFrameCommand.cs

```diff
@@ -33,16 +33,19 @@ namespace Microsoft.PythonTools.Repl {
     [ContentType(PythonCoreConstants.ContentType)]
     class DebugReplFrameCommand : IInteractiveWindowCommand {
         public Task<ExecutionResult> Execute(IInteractiveWindow window, string arguments) {
-            var eval = window.Evaluator as PythonDebugReplEvaluator;
-            if (eval != null) {
-                if (string.IsNullOrEmpty(arguments)) {
-                    eval.DisplayActiveFrame();
-                } else {
-                    int id;
-                    if (int.TryParse(arguments, out id)) {
-                        eval.ChangeActiveFrame(id);
+            var selectableEval = window.Evaluator as SelectableReplEvaluator;
```

Comment:
You can do `((window.Evaluator as SelectableReplEvaluator)?.Evaluator as PythonDebugReplEvaluator)` here, then you only need to do one explicit null check.